hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieBootstrapConfig.java

```diff
@@ -83,8 +79,8 @@ public class HoodieBootstrapConfig extends HoodieConfig {
   public static final ConfigProperty<String> KEYGEN_TYPE = ConfigProperty
       .key("hoodie.bootstrap.keygen.type")
       .defaultValue(KeyGeneratorType.SIMPLE.name())
-      .sinceVersion("0.9.0")
-      .withDocumentation("Type of build-in key generator, currently support SIMPLE, COMPLEX, TIMESTAMP, CUSTOM, NON_PARTITION, GLOBAL_DELETE");
+      .withEnumDocumentation(KeyGeneratorType.class, "Key generator class for bootstrap")
```

Comment:
For the second argument, is the convention to add a period (`.`) at the end or not?  I see both in different enum configs.

Reply:
Also, should all the descriptions be in the `@EnumDescription`, instead of having a second argument here for additional description?

Reply:
Sometimes the enum and the config don't match up completely.